crates/fluvio-connector-package/Cargo.toml

```diff
@@ -14,7 +14,7 @@ default = ["toml"]
 anyhow = { workspace = true }
 bytesize = { workspace = true }
 humantime-serde = { workspace = true }
-minijinja = { version = "2.6", default-features = false, features = ["custom_syntax", "fuel"] }
+minijinja = { version = "2.6", default-features = false, features = ["custom_syntax", "fuel", "serde"] }
```

Comment:
why this new feature?

Reply:
to pass clippy. it's required for new version